pkg/controller/utils/elasticsearch.go

```diff
@@ -20,8 +20,10 @@ import (
 	"crypto/tls"
```

Comment:
I guess this is a bit nitpicky, but I don't think this has anything to do with elasticsearch so I don't think it should be in this file.

Reply:
Makes sense. Moved to a new file